ijanki/prometheus-bundle/0.9/config/routes/ijanki_prometheus.yaml

```diff
@@ -0,0 +1,3 @@
+ijanki_prometheus:
+    resource: "@IjankiPrometheusBundle/Resources/config/routing.yml"
+    prefix: /
```

Comment:
Is this needed? Either add a value that changes the default behavior or remove it.

Reply:
So the routing will be automatically loaded by default? Ok, I removed it.